OrbitQt/main.cpp

```diff
@@ -127,6 +127,7 @@ static outcome::result<void> RunUiInstance(
       absl::StrFormat("127.0.0.1:%d", ports.grpc_port);
 
   OrbitMainWindow w(app, std::move(options));
+  w.resize(1280, 720);
```

Comment:
Can you add a comment? Otherwise I would assume somebody will remove this sooner than later.

Reply:
Good point. Done.